net-fs/curlftpfs/curlftpfs-0.9.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="Filesystem for accessing FTP hosts"
+DESCRIPTION="A file system for accessing ftp hosts based on FUSE and \
+libcurl.
+As the FTP protocol is not very feature rich, this filesystem does not \
+fulfill every constraint of a real filesystem, but it should be usable \
+for simple tasks like copying and editing files.
+
+CurlFtpFS differentiates itself from other FTP filesystems because it \
+features:
+
+* SSLv3 and TLSv1 support
+* connecting through tunneling HTTP proxies
+* automatically reconnection if the server times out
+* transform absolute symlinks to point back into the ftp file system
+
+Usage on Haiku: First start the userland_fs server:
+	/system/servers/userlandfs_server curlftpfs &
+
+Then mount an ftp share:
+mount -t userlandfs -p 'curlftpfs -o option1,option2 \\
+	ftp://someserver.somewhere/some/directory' /local/mount/directory
+
+Possible options are listed in the man page (includes SSL, proxy etc support)."
+HOMEPAGE="http://curlftpfs.sourceforge.net/"
+COPYRIGHT="2006-2008 Robson Braga Araujo"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/curlftpfs/curlftpfs-$portVersion.tar.gz"
+CHECKSUM_SHA256="4eb44739c7078ba0edde177bdd266c4cfb7c621075f47f64c85a06b12b3c6958"
+PATCHES="curlftpfs-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        curlftpfs$secondaryArchSuffix = $portVersion
+	addon:curlftpfs$secondaryArchSuffix
+        cmd:curlftpfs$secondaryArchSuffix
+        "
+REQUIRES="
+        haiku$secondaryArchSuffix
+        lib:libglib_2.0$secondaryArchSuffix
+        lib:libcurl$secondaryArchSuffix
+        gettext$secondaryArchSuffix
+        lib:libintl$secondaryArchSuffix
+        userland_fs
+        "
+
+BUILD_REQUIRES="
+        haiku${secondaryArchSuffix}_devel
+        devel:libglib_2.0$secondaryArchSuffix
+        userland_fs
+        devel:libcurl$secondaryArchSuffix
+        gettext$secondaryArchSuffix
+        gettext${secondaryArchSuffix}_libintl
```

Comment:
devel:libintl$secondaryArchSuffix should be enough, cmd:gettext$secondaryArchSuffix might go in BUILD_PREREQUIRES (if any).